src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,63 @@
+package parser;
+
+import exception.DukeException;
+
+import java.util.ArrayList;
+
+public class Parser {
+    public static ArrayList<String> parse(String input) throws DukeException {
+        ArrayList<String> commands = new ArrayList<>();
+        input = input.trim();
+        String[] inputs = input.split(" ");
+        commands.add(inputs[0]);
+        switch (commands.get(0)) {
+        case "list":
+            break;
+        case "mark":
```

Comment:
Reminder: The explicit //Fallthrough comment should be included whenever there is a case statement without a break statement. [See here](https://se-education.org/guides/conventions/java/basic.html#layout)